Styles/ECOWAS_Capacitor_Points_GCS_WGS84_WAPP_20230609.sld

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<StyledLayerDescriptor xmlns="http://www.opengis.net/sld" xmlns:se="http://www.opengis.net/se" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ogc="http://www.opengis.net/ogc" xmlns:xlink="http://www.w3.org/1999/xlink" xsi:schemaLocation="http://www.opengis.net/sld http://schemas.opengis.net/sld/1.1.0/StyledLayerDescriptor.xsd" version="1.1.0">
+  <NamedLayer>
+    <se:Name>ECOWAS_Capacitor_Points_GCS_WGS84_WAPP</se:Name>
+    <UserStyle>
+      <se:Name>ECOWAS_Capacitor_Points_GCS_WGS84_WAPP</se:Name>
+      <se:FeatureTypeStyle>
+        <se:Rule>
+          <se:Name>Single symbol</se:Name>
+          <se:PointSymbolizer>
+            <se:Graphic>
+              <!--Parametric SVG-->
+              <se:ExternalGraphic>
+                <se:OnlineResource xlink:type="simple" xlink:href="base64:PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz48IS0tIFVwbG9hZGVkIHRvOiBTVkcgUmVwbywgd3d3LnN2Z3JlcG8uY29tLCBHZW5lcmF0b3I6IFNWRyBSZXBvIE1peGVyIFRvb2xzIC0tPg0KPHN2ZyBmaWxsPSIjMDAwMDAwIiB3aWR0aD0iODAwcHgiIGhlaWdodD0iODAwcHgiIHZpZXdCb3g9IjAgMCAyNCAyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4NCjxwYXRoIGQ9Ik0yMiAxMUgxNC42QzE0LjcgOS40IDE1LjEgNy45IDE1LjggNi40TDE0IDUuNUMxMiA5LjUgMTIgMTQuNCAxNCAxOC40TDE1LjggMTcuNUMxNS4xIDE2LjEgMTQuNyAxNC41IDE0LjYgMTIuOUgyMlYxMVoiLz4NCjxwYXRoIGQ9Ik05IDExSDJWMTNIOVYxOUgxMVY1SDlWMTFaIi8+DQo8L3N2Zz4=?fill=%23ff0000&amp;fill-opacity=1&amp;outline=%23232323&amp;outline-opacity=1&amp;outline-width=0"/>
+                <se:Format>image/svg+xml</se:Format>
+              </se:ExternalGraphic>
+              <!--Plain SVG fallback, no parameters-->
+              <se:ExternalGraphic>
+                <se:OnlineResource xlink:type="simple" xlink:href="base64:PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz48IS0tIFVwbG9hZGVkIHRvOiBTVkcgUmVwbywgd3d3LnN2Z3JlcG8uY29tLCBHZW5lcmF0b3I6IFNWRyBSZXBvIE1peGVyIFRvb2xzIC0tPg0KPHN2ZyBmaWxsPSIjMDAwMDAwIiB3aWR0aD0iODAwcHgiIGhlaWdodD0iODAwcHgiIHZpZXdCb3g9IjAgMCAyNCAyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4NCjxwYXRoIGQ9Ik0yMiAxMUgxNC42QzE0LjcgOS40IDE1LjEgNy45IDE1LjggNi40TDE0IDUuNUMxMiA5LjUgMTIgMTQuNCAxNCAxOC40TDE1LjggMTcuNUMxNS4xIDE2LjEgMTQuNyAxNC41IDE0LjYgMTIuOUgyMlYxMVoiLz4NCjxwYXRoIGQ9Ik05IDExSDJWMTNIOVYxOUgxMVY1SDlWMTFaIi8+DQo8L3N2Zz4="/>
```

Comment:
Same here, rather export the svg as save it to the style folder and use something like
```
<PointSymbolizer>
2    <Graphic>
3      <ExternalGraphic>
4        <OnlineResource xlink:type="simple" xlink:href="file:///geoserver-data/burg02.svg" />
5        <Format>image/svg+xml</Format>
6      </ExternalGraphic>
7      <Size>20</Size>
8    </Graphic>
9  </PointSymbolizer>
```